kafka_consumer/datadog_checks/kafka_consumer/client.py

```diff
@@ -152,6 +154,9 @@ def get_highwater_offsets(self, consumer_offsets):
         return highwater_offsets
 
     def get_partitions_for_topic(self, topic):
+        if self.topics_cache.get(topic):
+            return self.topics_cache.get(topic)
```

Comment:
At first sight, I thought we were caching topics, but we are caching partitions. Could we rename this attribute to make it a bit more explicit?

Reply:
also nit: we could store the result of the very first `self.topics_cache.get(topic)` in a variable to avoid calling it twice

Reply:
Updated 👍